io.github.bobsonuk.edit_friendly_format.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app-id": "io.github.bobsonuk.edit_friendly_format",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "edit_friendly_format_main.sh",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=xdg-videos:create",
+    "--filesystem=xdg-music:create",
+    "--filesystem=home:ro"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "zenity",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/zenity/4.1/zenity-4.1.90.tar.xz",
+          "sha256": "bf3679c6205ff48dcebd1e1dff3a3a4a6a0b3a53e1cbc3b098a9ec0b62bd7236"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "cleanup": ["/include", "/lib/pkgconfig", "/share/ffmpeg/examples"],
+      "config-opts": [
+        "--enable-shared",
+        "--disable-static",
+        "--enable-gpl",
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-manpages"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ffmpeg.org/releases/ffmpeg-7.1.tar.xz",
+          "sha256": "40973d44970dbc83ef302b0609f2e74982be2d85916dd2ee7472d30678a7abe6"
+        }
+      ]
+    },
+    {
+      "name": "edit_friendly_format",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 edit_friendly_format_main.sh /app/bin/edit_friendly_format_main.sh",
+        "install -Dm755 edit_friendly_format_video.sh /app/bin/edit_friendly_format_video.sh",
+        "install -Dm755 edit_friendly_format_audio.sh /app/bin/edit_friendly_format_audio.sh",
+        "install -Dm644 io.github.bobsonuk.edit_friendly_format.desktop /app/share/applications/io.github.bobsonuk.edit_friendly_format.desktop",
+        "install -Dm644 io.github.bobsonuk.edit_friendly_format.svg /app/share/icons/hicolor/scalable/apps/io.github.bobsonuk.edit_friendly_format.svg",
+        "install -Dm644 io.github.bobsonuk.edit_friendly_format.metainfo.xml /app/share/metainfo/io.github.bobsonuk.edit_friendly_format.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/bobsonuk/edit_friendly_format/releases/download/v1.0.0/edit_friendly_format-1.0.0.tar.gz",
```

Comment:
this essentially seems to be 3 shell scripts https://github.com/BobsonUK/edit_friendly_format/blob/db35c6b8a9746f1e79cb4d6ad4f5c16aca1c1f91/src/edit_friendly_format_audio.sh


and as such we cannot accept this, see the policy at https://docs.flathub.org/docs/for-app-authors/requirements#submission-policy